ludwig/utils/strings_utils.py

```diff
@@ -282,31 +283,49 @@ def __init__(self, **kwargs):
     def __call__(self, text):
         pass
 
-
 class CharactersToListTokenizer(BaseTokenizer):
     def __call__(self, text):
         return text
 
-
 class SpaceStringToListTokenizer(BaseTokenizer):
     def __call__(self, text):
         return SPLIT_REGEX.split(text.strip())
 
+    @staticmethod
+    def call(t):
+        # TODO(ksbrar)
+        splitter = tf_text.RegexSplitter(split_regex=r'\s+')
```

Comment:
We should reuse the constant variables above to avoid duplication. We need probably convert them to back to strings.

If there's no way to go from regex to string, then we can separate them out:

```
SPLIT_REGEX_PATTERN = r'\s+'
SPLIT_REGEX = re.compile(SPLIT_REGEX_PATTERN)
```